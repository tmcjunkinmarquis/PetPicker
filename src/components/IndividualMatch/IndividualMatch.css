.individualMatch {
  width: 300px;
  height: 200px;
  border: 1px solid darkgray;
  color: blue;
}

.avatar-pic {
  width: 85px;
  height: 85px;
  padding: 0.3em;
}

.pet-title {
  font-weight: bold;
}
