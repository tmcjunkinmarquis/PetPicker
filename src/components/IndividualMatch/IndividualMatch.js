import React from 'react';
// import './IndividualMatch.css';
import PropTypes from 'prop-types';


const TestImportPath= (props) => {

  
  return (
    <div className='individual-match'>
      I am an individual match
    </div>
  );
};

export default TestImportPath;

TestImportPath.propTypes = {
  
};