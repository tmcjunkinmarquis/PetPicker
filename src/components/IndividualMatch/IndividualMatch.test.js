/* eslint-disable max-len */

import React from 'react';
import { shallow } from 'enzyme';
import IndividualMatch from './IndividualMatch';

describe('IndividualMatch', () => {
  let wrapper;
  let mockProps;

  beforeEach(() => {
    jest.resetAllMocks();
    mockProps = {};
    wrapper = shallow(<IndividualMatch {...mockProps} />, {
      disableLifecycleMethods: true
    });
  });

  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
