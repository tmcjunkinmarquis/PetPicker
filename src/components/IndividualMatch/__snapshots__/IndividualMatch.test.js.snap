// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndividualMatch matches the snapshot 1`] = `undefined`;
